test/absinthe/schema/rule/object_must_implement_interfaces_test.exs

```diff
@@ -22,6 +23,10 @@ defmodule Absinthe.Schema.Rule.ObjectMustImplementInterfacesTest do
     interface :parented do
       field :parent, :named
       field :another_parent, :named
+
+      resolve_type fn
+        _, _ -> nil
```

Comment:
Is there any reason why the resolve_type does not include `dog`, `cat` or `user` as in the other interfaces ?
Like:
```
resolve_type fn
  %{type: :dog}, _ -> :dog
  %{type: :user}, _ -> :user
  %{type: :cat}, _ -> :cat
  _, _ -> nil
end
```

Reply:
Forgot about that one 🙂

Reply:
Hehe no worry, as soon as you get the chance to fix this I will merge this PR, thanks again.

Reply:
👍